doc/projects.md

```diff
@@ -8,3 +8,6 @@ Add your project to this file and submit a pull request.
 4. [axe-firefox-devtools](https://github.com/dequelabs/axe-firefox-devtools) and on the [Firefox extension page](https://addons.mozilla.org/en-US/firefox/addon/axe-firefox-devtools/)
 5. [axe-selenium-java](https://github.com/dequelabs/axe-selenium-java)
 6. [a11yChromePlugin - not the official Chrome plugin source code](https://github.com/ptrstpp950/a11yChromePlugin)
+7. [grunt-axe-webdriver](https://www.npmjs.com/package/grunt-axe-webdriver)
+8. [R-Spec](https://github.com/dequelabs/axe-matchers)
+9. [Cucumber](https://github.com/dequelabs/axe-matchers)
```

Comment:
R-Spec and Cucumber probably shouldn't be on separate lines as they are both the same destination.